tidysic/settings/formatted_string.py

```diff
@@ -43,7 +44,7 @@ def write(self, taggable: Taggable) -> str:
                 self.text_after
             ))
 
-    def get_value(self, taggable: Taggable):
+    def get_value(self, taggable: Taggable) -> Union[str, int]:
```

Comment:
Shouldn't this be`Optional[Untion[str, int]]`? since the `getattr` can return `None` by default

Reply:
Turned this around a bit. Now it's `Optional[str]`.

Reply:
Nevermind, it's jusr `str`.